status-im/status-legacy
# Remove ens name from local database

fixes #8497

### Summary
Added option to remove ENS name from phone.

### Design notes
Removing name doesn't require contract transactions, so existing [figma designs](https://www.figma.com/file/TNCyHKtR3sx5EL6YznFWUa4O/Profile?node-id=4066%3A7399) not fully applicable.

#### Platforms
- Android
- iOS

### Steps to test
- Open Status
- remove one of previously added ens names
- make sure it is not present among available ones

status: ready
